command/plugin_list.go

```diff
@@ -40,13 +42,30 @@ Usage: vault plugin list [options] [TYPE]
 
       $ vault plugin list database
 
+  List all available plugins with detailed output:
+
+      $ vault plugin list -detailed
+
 ` + c.Flags().Help()
 
 	return strings.TrimSpace(helpText)
 }
 
 func (c *PluginListCommand) Flags() *FlagSets {
-	return c.flagSet(FlagSetHTTP | FlagSetOutputFormat)
+	set := c.flagSet(FlagSetHTTP | FlagSetOutputFormat)
+
+	f := set.NewFlagSet("Command Options")
+
+	f.BoolVar(&BoolVar{
+		Name:    "detailed",
+		Target:  &c.flagDetailed,
+		Default: false,
+		Usage: "Print detailed plugin information such as plugin type and " +
```

Comment:
I think there's a noun missing at the end of this line/start of the next? Or maybe the "and" should be deleted?